app/views/content/life-as-a-teacher/pay-and-benefits/teacher-pay.md

```diff
@@ -4,8 +4,9 @@ heading: "Teacher pay"
 description: |-
   All qualified teachers will have a starting salary of at least $salaries_starting_min$ (or higher in London). Find out about teacher pay ranges and more benefits of teaching.
 backlink: "../../"
-promo_content:
-  - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
+cta_mailinglist:
+  mailinglist:
+    text: Explore more benefits of teaching, and learn what it's really like to be in the classroom.
```

Comment:
Again, I know this is in prod but is there a better way to word this than `Explore`? That doesn't seem appropriate here?